firefox-ios/Client/Frontend/Browser/StartAtHomeHelper.swift

```diff
@@ -44,25 +44,19 @@ class StartAtHomeHelper: FeatureFlaggable {
     /// the user's last activity and whether, based on their settings, Start at Home feature
     /// should perform its function.
     public func shouldStartAtHome() -> Bool {
-        let setting = startAtHomeSetting
-        guard setting != .disabled else { return false }
-
-        let lastActiveTimestamp = UserDefaults.standard.object(forKey: "LastActiveTimestamp") as? Date ?? Date()
-        let dateComponents = Calendar.current.dateComponents([.hour, .second],
-                                                             from: lastActiveTimestamp,
-                                                             to: Date())
-        var timeSinceLastActivity = 0
-        var timeToOpenNewHome = 0
-
-        if setting == .afterFourHours {
-            timeSinceLastActivity = dateComponents.hour ?? 0
-            timeToOpenNewHome = 4
-        } else if setting == .always {
-            timeSinceLastActivity = dateComponents.second ?? 0
-            timeToOpenNewHome = 5
```

Comment:
please test this behaviour thoroughly. I remember there was a reason why this was done.... I just don't remember what that reason was. lololol

Reply:
I didn't know it, thanks @adudenamedruby for the reference. I'll run tests on it 💪

Reply:
Ok thanks @bmihai23 that you have restored it, so like this we comply to previous implementation.